index.html

```diff
@@ -1004,6 +1006,113 @@ <h3>
+              <var>protocol_handler</var>["url"] are undefined, [=iteration/continue=].
+              </li>
+              <li>Set <var>protocol_handler</var>["url"] to the result of [=URL
+              Parser|parsing=] <var>protocol_handler</var>["url"] using
+              <var>manifest URL</var> as the base URL. If the result is
+              failure, [=iteration/continue=].
+              </li>
+              <li>If <var>protocol_handler</var>["url"] is not
+              [=manifest/within scope=] of <var>manifest URL</var>, [=iteration/continue=].
+              </li>
+              <li>If <var>protocol_handler</var>["url"] already exists in <var>
+                processedProtocolHandlers</var>, [=iteration/continue=].
+              </li>
+              <li>
+                <a>Append</a> <var>protocol_handler</var> to
+                <var>processedProtocolHandlers</var>.
+              </li>
+            </ol>
+          </li>
+          <li>Return <var>processedProtocolHandlers</var>.
+          </li>
+        </ol>
+        <p>
+          To process the the <var>processedProtocolHandlers</var> the user
+          agent SHOULD [=register a protocol handler=] per item defined in the
+          [=sequence=].
+        </p>
+        <p>
+          A user agent SHOULD ask users for permission before registering a
+          <a>ProtocolHandlerItem</a> <var>protocol_handler</var>s as the
+          default handler for a protocol with the host operating system. A user
+          agent MAY truncate the list of <a>ProtocolHandlerItem</a>
+          <var>protocol_handlers</var> presented in order to remain consistent
+          with the conventions or limitations of the host operating system.
+        </p>
+        <div class="example">
+          <p>
+            In the following example, the developer has included two
+            <a>ProtocolHandlerItem</a> <var>protocol_handler</var>s. Assuming
+            the the manifest's URL is
+            <samp>https://example.com/manifest.webmanifest</samp>:
+          </p>
+          <ul>
+            <li>The first protocol handler would register to handle "web+music"
+            URLs (e.g.: web+music://#1234). When activated, the user agent
+            would instantiate a new <a>top-level browsing context</a> and
+            navigate to
+            <samp>https://example.com/play?songId=web+music://%231234</samp>.
+            </li>
+            <li>The second protocol handler would be ignored, as the protocol
+            provided does not start with "web+" and is not part of the
+            safelist.
+            </li>
+          </ul>
+          <pre class="example json">
+            {
+              "protocol_handlers": [
+                {
+                  "protocol": "web+music",
+                  "url": "/play?songId=%s"
+                },
+                {
+                  "protocol": "store",
+                  "url": "/buy?songId=%s"
```

Comment:
These are not in the [safe list](https://html.spec.whatwg.org/multipage/system-state.html#safelisted-scheme) ... so this won't work.

Reply:
yes indeed! the example states that it will be ignored as it does not comply with either starting with "Web+" or on the safelist

Reply:
Wait, I'm still confused. First, let's link "Safelist" here to HTML, just so that's clear. 

Secondly, "web+music" is not in the safe list of "[safelisted schemes](https://html.spec.whatwg.org/multipage/system-state.html#safelisted-scheme)"? So, it would still be rejected.

Reply:
Hola Marcos! So in this case "Web+music" would be accepted since the two accepted values for a protocol handler parameter during it's normalization are that: 

1. it is a value fomr the safelisted schemes you have share
2. it is a string that starts with "web+". This is the way to define 'custom' protocol handlers currently on the web and works with the `navigator.registerProtocolHandler()`.